lib/ret_web/controllers/api/v1/hub_controller.ex

```diff
@@ -45,6 +45,52 @@ defmodule RetWeb.Api.V1.HubController do
     end
   end
 
+  def update(conn, %{"id" => hub_sid, "hub" => hub_params}) do
+    account = Guardian.Plug.current_resource(conn)
+
+    case Hub |> Repo.get_by(hub_sid: hub_sid) |> Repo.preload([:created_by_account, :hub_bindings, :hub_role_memberships]) do
+      %Hub{} = hub -> 
+        if account |> can?(update_hub(hub)) do
+          update_with_hub(conn, account, hub, hub_params)
```

Comment:
nit: generally nicer to use the `conn |> ...` form where possible.

Reply:
Is it better here? We're not chaining any methods.

Reply:
After having gotten used to our elixir code, it's just a little surprising to see the traditional method call form, especially with operations related to `conn`s. More a matter of consistency than technical correctness. I also take it to mean that "we're piping a conn through this function and it likely returns a conn" as opposed to a method that produces some side effect and doesn't return a conn. Anyway, it was just a nit.